.github/workflows/delete-published-images.yml

```diff
@@ -34,7 +34,32 @@ on:
 #        - '^release-1\.\d+\.\d+$'
 
 jobs:
+  gate-check:
+    # Adding an `if:` that evaluates to false for this gate-check job prevents other dependent jobs from running.
+    runs-on: ubuntu-latest
+    steps:
+    - id: checkUserMember
+      # Only check for manual runs, not scheduled runs
+      if: github.event_name == 'workflow_dispatch'
+      uses: tspascoal/get-user-teams-membership@v2
+      with:
+        username: ${{ github.actor }}
+        team: 'vro-restricted'
+        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN_READ_TEAM }}
```

Comment:
Should this be setup as a reusable block?

Reply:
I considered that, but there's 2 different versions. Maybe someone can refactor that later.